File: atividade-2.py
def is_fibonacci(n):
    # Inicia os primeiros números da sequência
    a, b = 0, 1
    # Verifica se o número pertence à sequência
    while b <= n:
        if b == n:
            return True
        a, b = b, a + b
    return n == 0  # Caso especial para 0, que pertence à sequência

# Solicita ao usuário um número e verifica se ele pertence à sequência
numero = int(input("Informe um número: "))
if is_fibonacci(numero):
    print(f"O número {numero} pertence à sequência de Fibonacci.")
else:
    print(f"O número {numero} não pertence à sequência de Fibonacci.")

File: atividade-2.js
// Exercicio 2
function pertenceFibonacci(num) {
    // Iniciando os dois primeiros valores da sequência de Fibonacci
    let a = 0;
    let b = 1;

    // Caso o número seja 0 ou 1, já pertence à sequência
    if (num === 0 || num === 1) {
        return `O número ${num} pertence à sequência de Fibonacci.`;
    }

    // Calculando os próximos números da sequência até que b seja maior ou igual ao número informado
    while (b < num) {
        let temp = a + b;
        a = b;
        b = temp;
    }

    // Verifica se o número informado pertence à sequência
    if (b === num) {
        return `O número ${num} pertence à sequência de Fibonacci.`;
    } else {
        return `O número ${num} NÃO pertence à sequência de Fibonacci.`;
    }
}

// Exemplo de uso
let numero = 21; // Você pode alterar esse valor ou capturar a entrada do usuário
console.log(pertenceFibonacci(numero));